lxc/copy.go

```diff
@@ -4,33 +4,39 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.